automation/test.sh

```diff
@@ -377,7 +377,15 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ sig-monitoring ]]; then
     label_filter='(sig-monitoring)'
   elif [[ $TARGET =~ sig-operator ]]; then
-    label_filter='(sig-operator)'
+    if ! [[ $TARGET =~ sig-operator-upgrade ]]; then
```

Comment:
I am a bit confused by this logic, is it the same as this one?

```sh
    if [[ $TARGET =~ sig-operator-upgrade ]]; then
      label_filter='(sig-operator && Upgrade)'
    elif [[ $TARGET =~ sig-operator-configuration ]]; then
      label_filter='(sig-operator && !(Upgrade))'
    else
      label_filter='(sig-operator)'
    fi
```

Reply:
Done